src/Microsoft.DotNet.ProjectJsonMigration/MSBuildExtensions.cs

```diff
@@ -27,14 +27,18 @@ public static bool IsEquivalentTo(this ProjectItemElement item, ProjectItemEleme
             // Different includes
             if (item.IntersectIncludes(otherItem).Count() != item.Includes().Count())
             {
+#if !DISABLE_TRACE
```

Comment:
I introduced this since this class is internal and my tests are using it and MigrationTrace class is not relevant in the test case and is causing it to not compile.